packages/edit-post/src/components/header/template-title/delete-template.js

```diff
@@ -46,8 +46,7 @@ export default function DeleteTemplate() {
 		<MenuGroup className="edit-post-template-top-area__second-menu-group">
 			<MenuItem
 				isDestructive
-				isTertiary
-				isLink
+				variant="tertiary"
```

Comment:
Hey @jorgefilipecosta ,  we are currently in the process of replacing `isTertiary` and `isLink` with a new `variant` prop — which also means that a button can be assigned to only one variant at the time.

I've picked `tertiary` here, but I'd like to confirm with you this is the right choice here. Thank you!

Reply:
The design is changed a little bit. I think it is ok (provided we correct a small pixel alignment between the button and the input as a follow up):
new:
![detele button new](https://user-images.githubusercontent.com/11271197/119343957-ff53ec00-bc8e-11eb-84c2-55ec2eb3e253.jpg)


old:
![button-old](https://user-images.githubusercontent.com/11271197/119343983-05e26380-bc8f-11eb-825c-78f421b51e64.jpg)


But I'm going to cc: @jameskoster to double-check if we can do this UI change.

Reply:
I could also change it to `variant="link"` if that works better

Reply:
Should be fine, I actually have a [PR open](https://github.com/WordPress/gutenberg/pull/32042) to adjust that popover a bit. There it's only using `isSecondary` and `isDestructive` which I assume will work fine with the new system?

Reply:
> There it's only using isSecondary and isDestructive which I assume will work fine with the new system?

That's not going to be an issue. `isSecondary` will become `variant="secondary"` and `isDestructive` can stay as-is